mindbender/maya/plugins/collect_instances.py

```diff
@@ -61,6 +61,11 @@ def process(self, context):
                               objectsOnly=True):    # Return objectSet, rather
                                                     # than its members
 
+            # Check if objectset is empty
```

Comment:
In this case, no need to explain what this does. It should be obvious from the command itself.